client/src/components/new-request/analyze-results.vue

```diff
@@ -262,7 +262,7 @@ export default class AnalyzeResults extends Vue {
     this.originalOps = ops
   }
 
-  beforeDestroy () {
+  destroyed () {
```

Comment:
Both this and the previous Vue lifecycle methods are called at the end of this component's life. Everywhere else we use `destroyed` so I just made it consistent (and tested it -- it is correctly called to remove the event listener below).